.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Thanks for contributing to RIOT!
+
+### Preliminary checks
+
+The RIOT community cares a lot about code quality.
+Therefore before describing what your contribution is about, we would like you
+to check the following checklist:
+- [ ] Have you read the
+[coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions)
+of RIOT at least once?
+- [ ] Have you checked that your modifications are compliant with RIOT coding
+conventions?
+- [ ] Have you rebased your branch against latest RIOT `master` branch (
+`git pull --rebase upstream master`)
```

Comment:
it is "rebase on sth", not "rebase against" something. also, probably not everyone has "upstream" as remote, so the example command line might not work in that case.

Reply:
indeed, but I didn't know how to simply describe the procedure. I put the minimal list of commands in 'code' block, do you think it's better like this ?
Otherwise we could also add a 'check the github git usage documentation' ?